src/resources/views/configuration.blade.php

```diff
@@ -25,6 +24,7 @@
                                 @else
                                 <input type="text" class="form-control" id="teamspeak-configuration-hostname" name="teamspeak-configuration-hostname" value="{{ $tsHostname }}" />
                                 @endif
+                                <input type="hidden" name="_token" value="{{ csrf_token() }}" />
```

Comment:
best to use `csrf_field` helper, so we don't have to post an hotfix if framework change something in the way csrf is handled.